math/pronicnumber_test.go

```diff
@@ -0,0 +1,438 @@
+		{"n = 298", 298, false},
+		{"n = 299", 299, false},
+		{"n = 300", 300, false},
+		{"n = 301", 301, false},
+		{"n = 302", 302, false},
+		{"n = 303", 303, false},
+		{"n = 304", 304, false},
+		{"n = 305", 305, false},
+		{"n = 306", 306, true},
+		{"n = 307", 307, false},
+		{"n = 308", 308, false},
+		{"n = 309", 309, false},
+		{"n = 310", 310, false},
+		{"n = 311", 311, false},
+		{"n = 312", 312, false},
+		{"n = 313", 313, false},
+		{"n = 314", 314, false},
+		{"n = 315", 315, false},
+		{"n = 316", 316, false},
+		{"n = 317", 317, false},
+		{"n = 318", 318, false},
+		{"n = 319", 319, false},
+		{"n = 320", 320, false},
+		{"n = 321", 321, false},
+		{"n = 322", 322, false},
+		{"n = 323", 323, false},
+		{"n = 324", 324, false},
+		{"n = 325", 325, false},
+		{"n = 326", 326, false},
+		{"n = 327", 327, false},
+		{"n = 328", 328, false},
+		{"n = 329", 329, false},
+		{"n = 330", 330, false},
+		{"n = 331", 331, false},
+		{"n = 332", 332, false},
+		{"n = 333", 333, false},
+		{"n = 334", 334, false},
+		{"n = 335", 335, false},
+		{"n = 336", 336, false},
+		{"n = 337", 337, false},
+		{"n = 338", 338, false},
+		{"n = 339", 339, false},
+		{"n = 340", 340, false},
+		{"n = 341", 341, false},
+		{"n = 342", 342, true},
+		{"n = 343", 343, false},
+		{"n = 344", 344, false},
+		{"n = 345", 345, false},
+		{"n = 346", 346, false},
+		{"n = 347", 347, false},
+		{"n = 348", 348, false},
+		{"n = 349", 349, false},
+		{"n = 350", 350, false},
+		{"n = 351", 351, false},
+		{"n = 352", 352, false},
+		{"n = 353", 353, false},
+		{"n = 354", 354, false},
+		{"n = 355", 355, false},
+		{"n = 356", 356, false},
+		{"n = 357", 357, false},
+		{"n = 358", 358, false},
+		{"n = 359", 359, false},
+		{"n = 360", 360, false},
+		{"n = 361", 361, false},
+		{"n = 362", 362, false},
+		{"n = 363", 363, false},
+		{"n = 364", 364, false},
+		{"n = 365", 365, false},
+		{"n = 366", 366, false},
+		{"n = 367", 367, false},
+		{"n = 368", 368, false},
+		{"n = 369", 369, false},
+		{"n = 370", 370, false},
+		{"n = 371", 371, false},
+		{"n = 372", 372, false},
+		{"n = 373", 373, false},
+		{"n = 374", 374, false},
+		{"n = 375", 375, false},
+		{"n = 376", 376, false},
+		{"n = 377", 377, false},
+		{"n = 378", 378, false},
+		{"n = 379", 379, false},
+		{"n = 380", 380, true},
+		{"n = 381", 381, false},
+		{"n = 382", 382, false},
+		{"n = 383", 383, false},
+		{"n = 384", 384, false},
+		{"n = 385", 385, false},
+		{"n = 386", 386, false},
+		{"n = 387", 387, false},
+		{"n = 388", 388, false},
+		{"n = 389", 389, false},
+		{"n = 390", 390, false},
+		{"n = 391", 391, false},
+		{"n = 392", 392, false},
+		{"n = 393", 393, false},
+		{"n = 394", 394, false},
+		{"n = 395", 395, false},
+		{"n = 396", 396, false},
+		{"n = 397", 397, false},
+		{"n = 398", 398, false},
+		{"n = 399", 399, false},
+		{"n = 400", 400, false},
+		{"n = 2147441940", 2147441940, true},
+		{"n = 9223372033963249664", 9223372033963249664, true},
```

Comment:
`9223372033963249664` doesn't seem to be a pronic number. Please have a look [here](https://www.online-python.com/CxcDYaWjsZ).

Reply:
@vil02 I have found out that the largest 64-bit Pronic number is `9223372033963249500` and can be checked [here](https://www.online-python.com/npLgcyVFHG). 
I've push the related code fix and unit tests. Kindly have a look.

Thanks

Reply:
@itsAkshayDubey it seems to be correct now. Good job.